integration/skaffold/helper.go

```diff
@@ -346,7 +353,9 @@ func (b *RunBuilder) cmd(ctx context.Context) *exec.Cmd {
 	if value, found := os.LookupEnv("SKAFFOLD_BINARY"); found {
 		skaffoldBinary = value
 	}
-	b.env = append(b.env, "SKAFFOLD_INT_TEST=true")
+	if !b.cmdRunMode {
+		b.env = append(b.env, "SKAFFOLD_INT_TEST=true")
+	}
```

Comment:
Thanks @ericzzzzzzz for catching this. I didnt mean to merge this in.

Reply:
Lets revert this change `SKAFFOLD_INT_TEST=true` and then rebase to see if just reverting fixes the issue